### ISSUE TITLE
xw.view -  com_error: -2147024882, 'Not enough storage is available to complete this operation.'

### ISSUE THREAD

**Issue by y2kbugger** [xlwings/xlwings#730]
Receive  this error when trying to view a really large table.

Possibly related: #394 

xlwings 0.10.2

`C:\ProgramData\Anaconda3\lib\site-packages\xlwings\_xlwindows.py:1024: FutureWarning: to_datetime is deprecated. Use self.to_pydatetime()
`
```
return _datetime_to_com_time(x)
---------------------------------------------------------------------------
com_error                                 Traceback (most recent call last)
<ipython-input-19-0f9078fe5495> in <module>()
----> 1 xw.view(df)

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\main.py in view(obj, sheet)
   2675         sheet.clear()
   2676 
-> 2677     sheet.range('A1').value = obj
   2678     sheet.autofit()
   2679 

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\main.py in value(self, data)
   1493     @value.setter
   1494     def value(self, data):
-> 1495         conversion.write(data, self, self._options)
   1496 
   1497     def expand(self, mode='table'):

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\conversion\__init__.py in write(value, rng, options)
     33     pipeline = accessors.get(convert, convert).router(value, rng, options).writer(options)
     34     ctx = ConversionContext(rng=rng, value=value)
---> 35     pipeline(ctx)
     36     return ctx.value

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\conversion\framework.py in __call__(self, *args, **kwargs)
     64     def __call__(self, *args, **kwargs):
     65         for stage in self:
---> 66             stage(*args, **kwargs)
     67 
     68 

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\conversion\standard.py in __call__(self, ctx)
     89                     self._write_value(ctx.range[r:, :], ctx.value[r:], False)
     90             else:
---> 91                 self._write_value(ctx.range, ctx.value, scalar)
     92 
     93 

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\conversion\standard.py in _write_value(self, rng, value, scalar)
     69                 rng = rng.resize(len(value), len(value[0]))
     70 
---> 71             rng.raw_value = value
     72 
     73     def __call__(self, ctx):

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\main.py in raw_value(self, data)
   1114     @raw_value.setter
   1115     def raw_value(self, data):
-> 1116         self.impl.raw_value = data
   1117 
   1118     def clear_contents(self):

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\_xlwindows.py in raw_value(self, data)
    707     def raw_value(self, data):
    708         if self.xl is not None:
--> 709             self.xl.Value = data
    710 
    711     def clear_contents(self):

C:\ProgramData\Anaconda3\lib\site-packages\xlwings\_xlwindows.py in __setattr__(self, key, value)
     96         while True:
     97             try:
---> 98                 return setattr(self._inner, key, value)
     99             except pywintypes.com_error as e:
    100                 if (not N_COM_ATTEMPTS or n_attempt < N_COM_ATTEMPTS) and e.hresult == -2147418111:

C:\ProgramData\Anaconda3\lib\site-packages\win32com\client\dynamic.py in __setattr__(self, attr, value)
    563                                         entry = self._olerepr_.propMapPut[attr]
    564                                         invoke_type = _GetDescInvokeType(entry, pythoncom.INVOKE_PROPERTYPUT)
--> 565                                         self._oleobj_.Invoke(entry.dispid, 0, invoke_type, 0, value)
    566                                         return
    567                         try:

com_error: (-2147024882, 'Not enough storage is available to complete this operation.', None, None)
```

**Comment by stonebig** [comment]
maybe a COM limitation ? https://support.microsoft.com/en-us/help/890425/you-receive-a-not-enough-storage-is-available-to-complete-this-operati

**Comment by fzumstein** [comment]
This should be resolved by using `chunksize` introduced in 0.23.1
